include/rocksdb/iterator.h

```diff
@@ -108,6 +108,11 @@ class Iterator : public Cleanable {
   //   Get the user-key portion of the internal key at which the iteration
   //   stopped.
   virtual Status GetProperty(std::string prop_name, std::string* prop);
+
+  virtual Slice utimestamp() const {
```

Comment:
I guess "u" stands for "user"? Is "timestamp" good enough?

Reply:
Sadly no. class `TtlIterator` has a member function `uint32_t timestamp()` already.

Reply:
TtlIterator is an internal interface, which we can freely rename. Right?
This function may stay there forever, so it worth some time to come up with a good name.

Reply:
Sure, let me make the change and include in this PR.